pkg/telemetryquerylanguage/tql/functions_test.go

```diff
@@ -200,6 +200,35 @@ func Test_NewFunctionCall(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "getter slice arg",
+			inv: Invocation{
+				Function: "testing_getter_slice",
+				Arguments: []Value{
+					{
+						Path: &Path{
+							Fields: []Field{
+								{
+									Name: "name",
+								},
+							},
+						},
+					},
+					{
+						String: tqltest.Strp("test"),
+					},
+					{
+						Int: tqltest.Intp(1),
+					},
+					{
+						Float: tqltest.Floatp(1.1),
+					},
+					{
```

Comment:
Good test.  Can you add an arg that is an Invocation.

Reply:
Added to the test.